src/pyggdrasil/analyze/_metrics.py

```diff
@@ -7,11 +7,32 @@
 
 
 class Metrics:
-    """Metrics for comparing trees."""
+    """Metrics for comparing trees.
+
+    Attributes:
+        _METRICS: Dictionary of metrics.
+    """
 
     @staticmethod
     def get(metric: str) -> Callable[[TreeNode, TreeNode], float]:
-        """Return metric function."""
+        """Return metric function.
+
+        Args:
+            metric: Name of metric.
+
+        Returns:
+            - AD: Ancestor-Descendant Similarity;
+              pyggdrasil.distances.AncestorDescendantSimilarity().calculate,
+            - MP3: MP3 Similarity;
+              pyggdrasil.distances.MP3Similarity().calculate,
+            - TrueTree: True Tree Similarity;
+              pyggdrasil.compare_trees
+            - DL: Different Lineage Similarity;
+              pyggdrasil.distances.DifferentLineageSimilarity().calculate,
+            - MLTD: MLTD Similarity;
+              pyggdrasil.distances.MLTDSimilarity().calculate,
+
+        """
```

Comment:
Minor changes to documentation of metrics - I realized it was badly displayed in `mkdocs` otherwise